packages/loader/src/custom-file-loaders/Material.ts

```diff
@@ -0,0 +1,233 @@
+import { Engine, Material, ResourceManager, Shader, Texture } from "@oasis-engine/core";
+import { ShaderData } from "@oasis-engine/core/types/shader/ShaderData";
+import { Color, Matrix, Vector2, Vector3, Vector4 } from "@oasis-engine/math";
+import { parseRelativeUrl } from "../Util";
+import { BufferReader } from "./utils/BufferReader";
+
+export class MaterialDecoder {
```

Comment:
This file name is Material , duplicate with engine/core's Material, this is not good!
You can named this file with MaterialDecoder